models/kyberswap/base/kyberswap_aggregator_base_trades.sql

```diff
@@ -0,0 +1,112 @@
+        ,(meta_router.token_sold_amount_raw / power(10, (CASE meta_router.token_sold_address WHEN 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee THEN 18 ELSE p_sold.decimals END))) * (CASE meta_router.token_sold_address WHEN 0xeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee THEN  p_eth.price ELSE p_sold.price END)
+        )                                                               AS amount_usd
+    ,meta_router.token_bought_address                                   AS token_bought_address
+    ,meta_router.token_sold_address                                     AS token_sold_address
+    ,COALESCE(meta_router.taker, tx."from")                             AS taker
+    ,meta_router.maker                                                  AS maker
+    ,meta_router.project_contract_address                               AS project_contract_address
+    ,meta_router.tx_hash                                                AS tx_hash
+    ,tx."from"                                                          AS tx_from
+    ,tx."to"                                                            AS tx_to
+    ,meta_router.evt_index                                              AS evt_index
+    ,meta_router.trace_address                                          AS trace_address
+FROM meta_router
+INNER JOIN {{ source('base', 'transactions')}} tx
+    ON meta_router.tx_hash = tx.hash
+    {% if is_incremental() %}
+    AND {{incremental_predicate('tx.block_time')}}
+    {% else %}
+    AND tx.block_time >= TIMESTAMP '{{ project_start_date }}'
+    {% endif %}
+LEFT JOIN {{ source('tokens', 'erc20') }} erc20a
+    ON erc20a.contract_address = meta_router.token_bought_address
+    AND erc20a.blockchain = 'base'
+LEFT JOIN {{ source('tokens', 'erc20') }} erc20b
+    ON erc20b.contract_address = meta_router.token_sold_address
+    AND erc20b.blockchain = 'base'
+LEFT JOIN {{ source('prices', 'usd') }} p_bought
+    ON p_bought.minute = date_trunc('minute', meta_router.block_time)
+    AND p_bought.contract_address = meta_router.token_bought_address
+    AND p_bought.blockchain = 'base'
+    {% if is_incremental() %}
+    AND {{incremental_predicate('p_bought.minute')}}
+    {% else %}
+    AND p_bought.minute >= TIMESTAMP '{{ project_start_date }}'
+    {% endif %}
+LEFT JOIN {{ source('prices', 'usd') }} p_sold
+    ON p_sold.minute = date_trunc('minute', meta_router.block_time)
+    AND p_sold.contract_address = meta_router.token_sold_address
+    AND p_sold.blockchain = 'base'
+    {% if is_incremental() %}
+    AND {{incremental_predicate('p_sold.minute')}}
+    {% else %}
+    AND p_sold.minute >= TIMESTAMP '{{ project_start_date }}'
+    {% endif %}
+LEFT JOIN {{ source('prices', 'usd') }} p_eth
+    ON p_eth.minute = date_trunc('minute', meta_router.block_time)
+    AND p_eth.blockchain IS NULL
+    AND p_eth.symbol = 'ETH'
+    {% if is_incremental() %}
+    AND p_eth.minute >= date_trunc('day', now() - INTERVAL '7' DAY)
```

Comment:
update to new incremental predicate syntax plz

Reply:
@jeff-dude sorry I missed that line while updating the synthax, fixed now ser!